### DISCUSSION TITLE
dogecoind daemon crashing during download

### DISCUSSION THREAD

**Discussion by Jahzwolf** [dogecoin/dogecoin#2952] [Help needed]
Any ideas?  When I restart it seems to start over but moves fast is it validates what it has so far or what?

Hope it doesn't crash once the download is finished

**Comment by patricklodder** [comment]
Hi! I have some further questions to better understand what you're seeing.

1. With "download", do you mean initial block download / synchronization? 
2. What OS do you run on?
3. Do you use dogecoind or the graphical client?
4. What do you restart? The application or your PC?

**Comment by InTimeTechnologies** [comment]
Check your storage size. Perhaps it is running out of storage space and returns an error it cannot handle. They blockchain is approximately 86 Gb. If so, consider more storage space or running a pruned node.

**Comment by georgeartem** [comment]
have you been able to find documentation on how to run a pruned node?
